dev/README.md

```diff
@@ -49,7 +51,8 @@ gcloud auth configure-docker <REGISTRY_REGION>-docker.pkg.dev
 ### 2. Run tilt
 1. Set the path to the local git repo for [everest-operator](https://github.com/percona/everest-operator)
```

Comment:
```suggestion
1. Set the path to the local git repo for [everest-operator](https://github.com/percona/everest-operator) and [percona-helm-charts](https://github.com/percona/percona-helm-charts)
```

Reply:
Okay you already fixed this by changing the wording, your current solution is fine. I'm resolving this comment.